presto-main/src/test/java/com/facebook/presto/catalogserver/TestCatalogServerResponse.java

```diff
@@ -60,28 +60,32 @@ public void setup()
     public void testSchemaExists()
             throws Exception
```

Comment:
There is no exception in this test right?

Reply:
Think there was at one point but now there isnt any change of it, will remove